Sources/ApolloAPI/GraphQLEnum.swift

```diff
@@ -34,6 +34,10 @@ public enum GraphQLEnum<T: EnumType>: CaseIterable, Equatable, RawRepresentable
     self = .case(caseValue)
   }
 
+  public init(_ rawValue: String) {
```

Comment:
Should we just make this the designated initializer signature instead of having two public methods that do the same thing?

Reply:
I might be misunderstanding you, but `RawRepresentable` requires you to have a `public init(rawValue: String)` initializer, so we can't do that.